Tools/wpe/jhbuild.modules

```diff
@@ -69,12 +69,11 @@
 
   <meson id="glib"
          mesonargs="-Dlibmount=disabled -Dselinux=disabled">
-    <dependencies>
-      <dep package="libffi"/>
-    </dependencies>
-    <branch module="/sources/glib/2.70/glib-${version}.tar.xz" version="2.70.0"
+    <pkg-config>glib-2.0.pc</pkg-config>
+    <branch module="/sources/glib/2.70/glib-${version}.tar.xz"
```

Comment:
I think module should be 2.71: https://download.gnome.org/sources/glib/2.71/glib-2.71.3.tar.xz